howto/anbox/develop-platform.md

```diff
@@ -0,0 +1,84 @@
+Anbox Cloud provides a platform SDK which allows the development of custom platform plugins for the Anbox runtime.
+
+This guide assumes all steps are being run on a machine running Ubuntu 22.04 and hosting the [Anbox Cloud Appliance](https://discourse.ubuntu.com/t/install-appliance/22681).
+
+A platform module must be build on the same version of Ubuntu as the Anbox runtime. This means if you're using one of the Ubuntun 22.04 based Anbox images (e.g. `jammy:android12:arm64`) you must build on Ubuntu 22.04.
```

Comment:
```suggestion
A platform module must be built on the same version of Ubuntu as the Anbox runtime. This means if you're using one of the Anbox images based on Ubuntu 22.04 (for example, `jammy:android12:arm64`), you must build on Ubuntu 22.04.
```
I guess that means the Anbox runtime runs inside the Ubuntu container? Adding that might make it easier to understand; I had to read this three times to understand that we're talking about the images first and then about the appliance in the next sentence.